### PR TITLE
Provide extra details on claims processing

### PR DESCRIPTION
**Pull request by azdagron**
This PR addresses a recent ambiguity brought up in #182 (thanks, @rturner3!). It attempts to add clarifying language that the implementation must process the claims and return them to the client, but may selectively prune that list.

### CODE REVIEW

File: standards/SPIFFE_Workload_API.md [hunk 1]
```diff
@@ -311,7 +311,7 @@ message ValidateJWTSVIDResponse {
     // Required. The SPIFFE ID of the validated JWT-SVID.
     string spiffe_id = 1;
 
-    // Optional. Arbitrary claims contained within the payload of the validated
+    // Required. Arbitrary claims contained within the payload of the validated
     // JWT-SVID.
```
Comment (spikecurtis) [inline standards/SPIFFE_Workload_API.md]:
```suggestion
    // Required. Claims contained within the payload of the validated JWT-SVID.
    // This includes both SPIFFE-required and non-required claims.
```

**Review by spikecurtis — CHANGES_REQUESTED** [review]

File: standards/SPIFFE_Workload_API.md (hunk 1 above)
Comment (azdagron) [reply standards/SPIFFE_Workload_API.md]: :+1:

**Review by evan2645 — DISMISSED** [review]

File: standards/SPIFFE_Workload_API.md [hunk 2]
```diff
@@ -347,7 +347,7 @@ As mentioned in [Stream Responses](#42-stream-responses), each `JWTBundleRespons
 
 #### 6.2.3 ValidateJWTSVID
 
-The `ValidateJWTSVID` RPC validates JWT-SVIDs for a specific audience on behalf of a client.
+The `ValidateJWTSVID` RPC validates JWT-SVIDs for a specific audience on behalf of a client. Further, the server MUST parse and validate the JWT-SVID according to the rules outlined in the [JWT-SVID](JWT-SVID.md) specification. The claims embedded in the JWT-SVID payload MUST be provided in the `claims` field in the `ValidateJWTSVIDResponse`, however implementations MAY filter those claims before returning them to the client.
```
Comment (justinburke) [inline standards/SPIFFE_Workload_API.md]:
```suggestion
The `ValidateJWTSVID` RPC validates JWT-SVIDs for a specific audience on behalf of a client. Further, the server MUST parse and validate the JWT-SVID according to the rules outlined in the [JWT-SVID](JWT-SVID.md) specification. The claims embedded in the JWT-SVID payload SHOULD be provided in the `claims` field in the `ValidateJWTSVIDResponse`; the claims defined by this specification above are required, however implementations MAY filter non-SPIFFE claims before returning them to the client. SPIFFE claims are required for interoperability.
```

**Review by spikecurtis — COMMENTED** [review]
LGTM with Justin's suggested wording change.

**Comment by azdagron** [comment]
Awesome. I've made Justin's suggested edits.

**Review by evan2645 — APPROVED** [review]

**Review by spikecurtis — APPROVED** [review]

**Pull request merged**